tests/logs/delete/test_delete_packages/test_delete_files.txt

```diff
@@ -0,0 +1,23 @@
+[    INFO] Get files: started
+[ WARNING] Requested unit(s) doesn't exist as file: random.txt
+[    INFO] 3 unit(s) found for deletion
+[    INFO] Get files: finished
+[ WARNING] other.iso is not present in some-filerepo
+[    INFO] Deleting hello.iso from ['some-filerepo']
```

Comment:
Can we please avoid default string formatting of list in all these logs, and instead join them?  So it'd say "Deleting hello.iso from repo1, repo2" instead of "Deleting hello.iso from ['repo1', 'repo2']".  The string formatting of a list seems developer-oriented.

Reply:
Done